packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -44,6 +44,7 @@ const CompleteResetPasswordContainer = ({
 
   if (!location.state) {
     navigate('/reset_password', { replace: true });
+    return;
```

Comment:
I think we're inconsistent about what we return, I see returns like this and `return <LoadingSpinner fullScreen />;` and returning a React fragment so the return type is always a React element. Not sure if users would see the loading spinner here, so 🤷‍♀️ r+

Reply:
Yes, I was wondering about this. It also occurred to me that there might be more of these scenarios, but I was able to validate that the navigate happens anyway, and the error wasn't impacting the navigation. I think this state after navigation is really just a blip.